src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketsTable.tsx

```diff
@@ -74,18 +75,14 @@ const AnnotationTicketsTable: React.VFC<AnnotationTicketsProps> = ({
             condition: userCanModify,
             wrapper: (children: JSX.Element) => (
               <Popconfirm
```

Comment:
these buttons don't seem to be working for me either 😭

also nit. perhaps we should update the button text from `Move To Suspected Issues` to `Move to suspected issues`